dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -21,15 +21,48 @@
          ibmui:localization="OSGI-INF/l10n/metatype">
                               
         <!--  enableImplicitBeanArchives means that the CDI runtime will scan bean-definition annotations for the archives that do not contain beans.xml -->
+        <!--  defaults to true but this is now done in code rather than configuration -->
         <AD id="enableImplicitBeanArchives"
                               name="%cdi12.enableImplicitBeanArchives.name"
                               description="%cdi12.enableImplicitBeanArchives.desc"
-                              required="false" type="Boolean" default="true" />
+                              required="false" type="Boolean" />
 
     </OCD>
   
     <Designate pid="com.ibm.ws.cdi12.cdiContainer">
         <Object ocdref="com.ibm.ws.cdi12.cdiContainer"/>
     </Designate>
     
+    <!-- <OCD name="%cdi" 
+         description="%cdi.desc" -->
+    <OCD name="internal"
+         description="internal use only"
+         id="io.openliberty.cdi.configuration" 
+         ibm:alias="cdi"
+         ibmui:localization="OSGI-INF/l10n/metatype">
+                              
+        <!--  enableImplicitBeanArchives means that the CDI runtime will scan bean-definition annotations for the archives that do not contain beans.xml -->
+        <!--  defaults to true but this is now done in code rather than configuration -->
+        <!-- <AD name="%cdi.enableImplicitBeanArchives.name"
+                 description="%cdi.enableImplicitBeanArchives.desc" -->
+        <AD name="internal"
+            description="internal use only"
+            id="enableImplicitBeanArchives"
+            required="false" type="Boolean" />
+                              
+        <!--  emptyBeansXMLExplicitArchive will cause an archive with an empty beans.xml to be considered an explicit bean archive. -->
+        <!--  defaults to false but this is now done in code rather than configuration -->
```

Comment:
```suggestion
        <!--  emptyBeansXMLExplicitArchive will cause an archive with an empty beans.xml to be considered an explicit bean archive. -->
        <!--  defaults to false but this is done in code rather than configuration -->
        <!--  only applies to cdi-4.0 and newer -->
```